ui/vty/src/Ariadne/UI/Vty/Widget/Status.hs

```diff
@@ -23,46 +23,48 @@ import Ariadne.UI.Vty.Face
 
 data StatusWidgetState =
   StatusWidgetState
-    { statusWidgetTipHeaderHash :: !Text
-    , statusWidgetTipSlot :: !Text
-    , statusWidgetSlot :: !Text
+    { statusWidgetSyncProgress :: !(Maybe Text)
+    , statusWidgetBlockchainLocal :: !Text
+    , statusWidgetBlockchainNetwork :: !Text
     , statusWidgetNewVersion :: !(Maybe Version)
     }
 
 makeLensesWith postfixLFields ''StatusWidgetState
 
 initStatusWidget :: StatusWidgetState
 initStatusWidget = StatusWidgetState
-    { statusWidgetTipHeaderHash = "<unknown>"
-    , statusWidgetTipSlot = "<unknown>"
-    , statusWidgetSlot = "<unknown>"
+    { statusWidgetSyncProgress = Nothing
+    , statusWidgetBlockchainLocal = "<unknown>"
+    , statusWidgetBlockchainNetwork = "<unknown>"
     , statusWidgetNewVersion = Nothing
     }
 
 drawStatusWidget :: Text `Named` "ariadne_url" -> StatusWidgetState -> B.Widget name
-drawStatusWidget (Named ariadneURL) statusWidgetState =
+drawStatusWidget (Named ariadneURL) StatusWidgetState{..} =
   B.padTop (B.Pad 1) $
     B.updateAttrMap (B.mapAttrName "status" B.borderAttr) $
     B.withAttr "status" $
-    (case statusWidgetState ^. statusWidgetNewVersionL of
-      Just ver -> (B.<=> B.padLeftRight 1 (updateNotification ver))
-      Nothing -> identity
-    ) $
+    maybe identity (\ver -> (B.<=> updateNotification ver)) statusWidgetNewVersion $
```

Comment:
`B.(<=>) . updateNotification` ?

Reply:
Your code will put notification _above_ status bar (as first argument), not _below_ (as second argument)